upgrade/UPGRADE-v9.1.0-dev.md

```diff
@@ -249,3 +249,6 @@ There you can find links to upgrade notes for other versions too.
 
 - added total count to frontend API connections ([#2141](https://github.com/shopsys/shopsys/pull/2141))
     - see #project-base-diff to update your project
+
+- use DBAL native queries for DML instead of ORM Native SQL ([#2148](https://github.com/shopsys/shopsys/pull/2148))
+    - see #project-base-diff to update your project
```

Comment:
I am missing a hint about type defining. I belive it could be handy.

Reply:
Good point. added in b67e1b1be